lib/pacer_test.go

```diff
@@ -0,0 +1,148 @@
+package vegeta
+
+import (
+	"math"
+	"testing"
+	"time"
+)
+
+func TestRatePacing(t *testing.T) {
+	t.Parallel()
+
+	for i, tt := range []struct {
+		freq    int
+		per     time.Duration
+		elapsed time.Duration
+		count   uint64
+		wait    time.Duration
+		stop    bool
+	}{
+		// NOTE: Rate sleeps before sending the first hit,
```

Comment:
First off, Rate doesn't actually sleep, it just returns some `time.Duration` for the caller to do with it as he wishes. Thus this might be misleading for the reader.

Second, this is an implementation detail and is easily deduced from some of the test-cases below, so, I would say this comment is not necessary here at all.

Reply:
OK, fair point, I'll remove this.